sdk/containerregistry/container-registry/samples-dev/deleteImages.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Illustrates how to delete all but the latest three images.
```

Comment:
Nit: same comment as Will's in https://github.com/Azure/azure-sdk-for-js/pull/15023#discussion_r622374360

Copied to save you a click:

> Nit: these all ended up pretty same-y, and the older format is more concise and direct IMO. "uses", "creates", and "purges" are all strong active verbs, and "demonstrates" isn't as much. We found that when we tried to make all the sample summaries into full sentences rather than fragments, we always ended up with something like "This sample demonstrates" or "This sample shows" at the beginning, so typically I prefer the more concise form.